packages/engine-core/src/tasks/patch/taskPatchReset.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    RnvTaskName,
+    createTask,
+    fsExistsSync,
+    fsReadFileSync,
+    getConfigRootProp,
+    logToSummary,
+    logWarning,
+    removeDirSync,
+    revertOverrideToOriginal,
+} from '@rnv/core';
+import path from 'path';
+
+export default createTask({
+    description: 'Reset applied overrides',
+    fn: async ({ ctx }) => {
+        const isMonorepo = getConfigRootProp('isMonorepo');
+        const overrideDir = isMonorepo
+            ? path.join(ctx.paths.project.dir, '../../.rnv', 'overrides')
+            : path.join(ctx.paths.project.dir, '.rnv', 'overrides');
+
+        if (!fsExistsSync(overrideDir)) {
+            return logWarning(`Plugin overrides have not been applied yet`);
+        }
+        const rootPath = isMonorepo ? path.join(ctx.paths.project.dir, '../..') : path.join(ctx.paths.project.dir);
+        const appliedOverrideFilePath = path.join(overrideDir, 'applied_overrides.json');
```

Comment:
Is it expected that `patch reset` does not revert file-based overrides, only `overrides.json`? If so, the description should specify that, and it should maybe print a warning.